client/src/components/DataFiles/DataFilesProjectListing/DataFilesProjectListing.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DataFilesListing from '../DataFilesListing/DataFilesListing';
+
+const DataFilesProjectListing = ({ system, path }) => {
```

Comment:
Can this be `DataFilesProjectFileListing` instead to make it more apparent that we're listing files and not projects?
(or `DataFilesProjectDetail` since we're going to be showing project-specific metadata above the listing).